medio/backends/itk_io.py

```diff
@@ -36,11 +36,19 @@ def read_img(input_path, desired_axcodes=None, header=False, components_axis=Non
         :return: numpy image and metadata object which includes pixdim, affine, original orientation string and
         coordinates system
         """
+        if header:
+            imageio = itk.GDCMImageIO.New()
```

Comment:
What if `header=True` and the image is not a dicom? It would fail, right?
We should sniff the format first